### PR TITLE
[zombienet] add nominators

### PR DESCRIPTION
**Pull request by pepoviola**
Hi @ordian, changes to add nominators:
- bump zombienet.
- change network file to add 25000 nominators.
- set  `MaxElectingVoters` to `22_500` for fast-runtime.

Thanks!

### CODE REVIEW

File: runtime/kusama/src/lib.rs [hunk 1]
```diff
@@ -408,8 +408,8 @@ parameter_types! {
 	// 1 hour session, 15 minutes unsigned phase, 8 offchain executions.
 	pub OffchainRepeat: BlockNumber = UnsignedPhase::get() / 8;
 
-	/// We take the top 12500 nominators as electing voters..
-	pub const MaxElectingVoters: u32 = 12_500;
+	/// We take the top 12500/22500 nominators as electing voters..
+	pub const MaxElectingVoters: u32 = prod_or_fast!(12_500, 22_500);
```
Comment (ordian) [inline runtime/kusama/src/lib.rs:412]: this should either be 25k, or random_nominators_count below 22.5k

Comment (pepoviola) [reply runtime/kusama/src/lib.rs]: Updated, thanks!!

**Review by ordian — APPROVED** [review]

**Review by sandreim — APPROVED** [review]

File: runtime/kusama/src/lib.rs [hunk 2]
```diff
@@ -408,8 +408,8 @@ parameter_types! {
 	// 1 hour session, 15 minutes unsigned phase, 8 offchain executions.
 	pub OffchainRepeat: BlockNumber = UnsignedPhase::get() / 8;
 
-	/// We take the top 12500 nominators as electing voters..
-	pub const MaxElectingVoters: u32 = 12_500;
+	/// We take the top 12500/22500 nominators as electing voters..
+	pub const MaxElectingVoters: u32 = prod_or_fast!(12_500, 25_000);
```
Comment (kianenigma) [inline runtime/kusama/src/lib.rs:412]: This doesn't really make sense. If a zombinet is being launched based on Kusama, it should respect its limit. 25000 nominators, from which 12_500 being elected is reasonable. 

This won't help us test this for the scale of Polkadot though.

**Review by kianenigma — CHANGES_REQUESTED** [review]
We should not tweak kusama runtime in an unreasonable way here. Let's make the current test be only Kusama, and test with Polkadot's scale (25000) elsewhere.

File: runtime/kusama/src/lib.rs (hunk 2 above)
Comment (ordian) [reply runtime/kusama/src/lib.rs:412]: Note that this is one-time test and we're testing here the limits including allocation limits. So having more nominators than what's *currently* on Kusama seems reasonable to me.

Comment (pepoviola) [reply runtime/kusama/src/lib.rs:412]: Thanks for the clarification, I will revert this changes 👍

**Review by kianenigma — APPROVED** [review]
ahh, given the target branch, it is all good. Sorry 😜

**Pull request merged**